src/openms/source/FORMAT/OSWFile.cpp

```diff
@@ -672,7 +672,7 @@ namespace OpenMS
           Sql::extractFloat(stmt, COLIDs::PRODUCT_MZ),
           Sql::extractChar(stmt, COLIDs::TYPE),
           Sql::extractInt(stmt, COLIDs::DECOY));
-        swath_result.addTransition(std::move(tr));
+        swath_result.addTransition(tr);
```

Comment:
check?

Reply:
requires implementation
```suggestion
        swath_result.addTransition(tr); // TODO implement
```

Reply:
will address this is in #5685